doc/index.rst

```diff
@@ -23,71 +23,9 @@ Site contents
 .. toctree::
    :maxdepth: 2
 
+   introduction
+   preparing
    dockerfile
    reproducibility
    sample_repos
    faq
-
```

Comment:
Moved this into `introdcution.rst` because as a reader I rarely expect to find stuff below the TOC. I read the welcome blah above the TOC, then start scanning for what I am looking for, follow the link. Opinions?